### PR TITLE
Add monthly mode for slipped sprint items.

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes [#395](https://github.com/dotnet/docs-tools/issues/395)

Add a mode where all labeled open issues or pull requests are processed. This runs once on the morning after the 5th of the month to update any items slipped in the previous sprint.

Only move items to the "future" sprint after the 5th of the month.

Create workflow to run the new mode once a month.

### CODE REVIEW

File: .github/workflows/quest-all.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: "bulk quest import"
```
Comment (BillWagner) [inline .github/workflows/quest-all.yml:1]: @IEvangelist This is a copy-paste-modify from the `quest-bulk.yml` workflow.

If there's a way to integrate this into the regular `bulk` workflow, I want your suggestions.

**Review by adegeo — APPROVED** [review]

File: .github/workflows/quest-all.yml (hunk 1 above)
Comment (IEvangelist) [reply .github/workflows/quest-all.yml:1]: What's different between the two?

**Review by IEvangelist — APPROVED** [review]

File: .github/workflows/quest-all.yml (hunk 1 above)
Comment (BillWagner) [reply .github/workflows/quest-all.yml:1]: There are two differences:

The monthly version uses a duration of "-1". That instructs the code to walk all open issues with the configured labels. It also runs only once a month.

Is there a way to combine that so the schedule sets the parameters differently? E.g. every night, run with duration = 5, and once a month run with duration = -1?

Comment (BillWagner) [inline .github/workflows/quest-all.yml:1]:
```suggestion
name: "Monthly bulk quest import"
```

**Pull request merged**